shop/src/themes/shared/checkout/Information.js

```diff
@@ -134,8 +134,7 @@ export const MobileInformation = () => {
 
   const onSubmit = (e) => {
     e.preventDefault()
-
-    const { valid, newState } = validate(state)
+    const { valid, newState } = validate(state, ['email'])
```

Comment:
Hi @etnlbck,

I worked this line into the [function definition](https://github.com/OriginProtocol/dshop/blob/master/shop/src/data/validations/checkoutInfo.js) for the `validate` function. Here's what I found:
Let's say the user (i.e. the "customer") is on a mobile phone and lands on the checkout page in consideration. They input nothing in the 'Email' field and hit 'Continue'. When they do so, the system calls the function `validate(state, ['email'])`, with
<pre>state.email = undefined</pre>
Stepping through `validate`, we first find
<pre>
> newState = {}
> newState = { emailError: some_value, firstNameError: some_other_value, ....., zipError: yet_another_value}
> filteredFields = ['emailError', 'firstNameError', .... , 'zipError']
</pre>

Evaluating the`filter` function in line 78 results in :
<pre>
filteredFields = ['']
</pre>

Because of this, `valid` will be set to `true` and the system will allow the user to continue to the next form.

Conclusion: One way to address this is to update the line to say `validate(state, ['emailError']`.

Reply:
@phyninja great catch. I'll update.